02-social-links-profile/styles.css

```
@@ -0,0 +1,125 @@
+:root {
+  --Black: #141414;
+  --Dark-Grey: #1f1f1f;
+  --White: #fff;
+  --Neon-Green: #c4f82a;
+  --Grey: #333;
+}
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  display: flex;
+  width: 375px;
+  padding: 116px 24px 117px 24px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  background: var(--Black);
+}
+
+.tarjeta {
+  display: flex;
+  width: 327px;
+  padding: 24px;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  border-radius: 12px;
+  background: var(--Dark-Grey);
+}
+
+.tarjeta::before {
+  content: "";
+  display: block;
+  width: 88px;
+  height: 88px;
+  background-image: url("../02-social-links-profile/assets/images/avatar-jessica.jpeg");
+  background-size: cover;
+  background-position: center;
+  border-radius: 999px;
+}
+
+.tarje-texts {
+  text-align: center;
+  font-family: Inter;
+  line-height: 150%; /* 21px */
+}
+.tarje-title {
+  color: var(--White, var(--White));
```

Recuerda bien como se deben usar las variables en CSS
```suggestion
  color: var(--White);
```